frontend/src/lib/components/SaveToDashboard/SaveToDashboardModal.tsx

```diff
@@ -65,7 +61,7 @@ export function SaveToDashboardModal({
                 }
             }
         } else {
-            await api.update(`api/insight/${fromItem}`, { filters })
+            await api.update(`api/insight/${fromItem}`, { ...filters })
```

Comment:
Context on why we're destructuring filters now? Think the API endpoint still expected the `filters` attr

Reply:
you're right here, reverting this change